doc/commands/package.md

```diff
@@ -30,3 +31,28 @@ Run all available standard ADT checks for all objects of the give package.
 ```bash
 sapcli package check \$productive_code
 ```
+
+## stat
+
+Prints basic set of package attributes. Returned exit could be interpreted as:
+* `0` - package found
+* `1` - package not found
```

Comment:
Since you want to make assumptions based on exit codes, I would propose skipping 1 and 2 and use 3 or higher -> https://tldp.org/LDP/abs/html/exitcodes.html

Argh, `stat /foo` exits with 1.
And ENOENT is 2.
```
# grep -r ENOENT /usr/include/
/usr/include/asm-generic/errno-base.h:#define   ENOENT           2      /* No such file or directory */
```

Perhaps, we can have a common exit code for not found entity which can be used in other objects - e.g. "sapcli class read CL_DOES_NOT_EXIST".

Well, if you want to ignore my proposal, just let me know.

Reply:
Makes sense. I'll increase this number and define it as a constant to allow reuse.